test/suites/drift/suite_test.go

```diff
@@ -301,16 +276,14 @@ var _ = Describe("Drift", func() {
 			By("drifting the nodepool")
 			nodePool.Spec.Template.Annotations = lo.Assign(nodePool.Spec.Template.Annotations, map[string]string{"test-annotation": "drift"})
 			env.ExpectUpdated(nodePool)
-
-			// Ensure that we get three nodes tainted, and they have overlap during the drift
-			env.EventuallyExpectTaintedNodeCount("==", 3)
-			env.EventuallyExpectNodeClaimCount("==", 8)
-			env.EventuallyExpectNodeCount("==", 8)
-			env.ConsistentlyExpectDisruptionsWithNodeCount(3, 8, 5*time.Second)
+			env.ConsistentlyExpectDisruptionsUntilTarget(3, 5, 0, 10*time.Minute)
```

Comment:
If we're specifying 0 everywhere does it even need to be an argument?

Reply:
I think that's fair that we'll probably never stop disruptions halfway through in our tests since that's too tough to assert without being racey, good call